.github/scripts/commons.mjs

```diff
@@ -8,17 +8,20 @@ export const config = {
   testAssetsDir: `./test-assets/`,
 }
 
+export const branchName = process.env.BRANCH_NAME || 'main';
+
 export async function cloneWcagActRules({ tmpDir }) {
   await $`rm -rf ${tmpDir}`;
   await $`git clone \
     --no-single-branch \
     --branch main \
-    https://github.com/w3c/wcag-act-rules.git ${tmpDir} \
+    https://github.com/daniel-montalvo/wcag-act-rules.git ${tmpDir} \
     --depth 1
   `;
 }
 
 export async function createOrCheckoutBranch({ tmpDir }, branchName) {
+  branchName = process.env.branch_name || 'main'
   assert(branchName, 'branchName must be defined');
```

Comment:
* This does not use anymore the `branchName` parameter of the function, which looks weird.
* This is the same as the `const` defined on line 11, so it can just re-use it directly.
* The assertion can be removed now that there is a default.
* (here also,  I prefer nullish over falsy coalescence)